benchmark-framework/src/main/java/io/openmessaging/benchmark/ProducerRateController.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.openmessaging.benchmark;
+
+public interface ProducerRateController {
```

Comment:
Could do with a mechanism to load configuration from driver/workload.

```
interface ProducerRateController {
  void init(Properties properties);
  double nextTargetProducerRate(...);
}
//or
interface ProducerRateController {
  void init(Properties properties);
  double nextTargetProducerRate(...);
  interface Factory {
    ProducerRateController create(Properties properties);
  }
}
```
or expect a constructor with Properties param?